clouddriver-aws/src/test/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/ops/loadbalancer/UpsertAmazonLoadBalancerClassicAtomicOperationSpec.groovy

```diff
@@ -86,21 +86,57 @@ class UpsertAmazonLoadBalancerClassicAtomicOperationSpec extends Specification {
   def regionScopedProviderFactory = Stub(RegionScopedProviderFactory) {
     forRegion(_, "us-east-1") >> regionScopedProvider
   }
+
+  def securityGroupLookup = Mock(SecurityGroupLookup)
```

Comment:
Out of curiosity, is there a test when the application security group does not exist? Looking at the code we'll create it right?

Reply:
Yes. we will create it. I will add a test for that as well